core/transport/src/grpc/block_worker.proto

```diff
@@ -123,14 +124,15 @@ message OpenLocalBlockResponse {
   optional string path = 1;
 }
 
-// next available id: 6
+// next available id: 8
 message CreateLocalBlockRequest {
   optional int64 block_id = 1;
   optional int32 tier = 3;
   optional int64 space_to_reserve = 4;
   // If set, only reserve space for the block.
   optional bool only_reserve_space = 5;
   optional bool cleanup_on_failure = 6;
+  optional string medium_type = 7;
```

Comment:
I would prefer to have a consistent name for this type of field, currently we have `writeMedium`, `mediumType`, and `pinnedMedia`.

Reply:
I changed `writeMedium` to  `mediumType`, I think `pinnedMedia` refers to a list of pinned medium specifically, so that one is kind of different?

Reply:
Yeah you are right, there are two concepts, singular medium type to write to and a list of media to pin to